Tests/Twig/Extension/SonataAdminExtensionTest.php

```diff
@@ -2002,6 +2130,24 @@ public function testGetUrlsafeIdentifier_GivenAdmin_Bar()
         $this->assertSame(1234567, $this->twigExtension->getUrlsafeIdentifier($entity, $this->adminBar));
     }
 
+    /**
+     * Remove this method for next major release only if PHP 5.3 support will be dropped
+     * This method generate url part for twig layout. Allow to keep BC for PHP 5.3.
```

Comment:
> 

    generate -> generates
    for twig -> for Twig
    Allow -> Allows

@Soullivaneuh Fixed